modules/local/generate_run_report/templates/generate_run_report.py

```diff
@@ -856,6 +856,23 @@ def primer_mismatch_heatmap(
         else len(primer_pairs)
     )
 
+# parsing of nextclade lineages.tsv output
+nextclade_rows = []
+nextclade_tsv = "nextclade.tsv"
+if Path(nextclade_tsv).is_file():
+    with open(nextclade_tsv, newline="") as fh:
+        for row in csv.DictReader(fh, delimiter="\t"):
+            nextclade_rows.append(
+                {
+                    "Sample": row.get("seqName", ""),
+                    "QC_status": row.get("qc_status", ""),
+                    "Clade": row.get("clade", ""),
+                    "Lineage": row.get("lineage", ""),
+                    "Cov": row.get("qc_overall_score", ""),
+                }
+            )
+payload["nextclade_table"] = nextclade_rows
```

Comment:
as far as I can tell this doesn't actually make it into the report itself (no changes to the template), ideally any new outputs should be included in a tab in the run report.